Makefile

```diff
@@ -57,16 +61,20 @@ dist/po.%.js: po/%.po $(NODE_MODULES_TEST)
 	po/po2json -m po/po.empty.js -o $@.js.tmp $<
 	mv $@.js.tmp $@
 
-ZANATA_CLI = LANG=C.UTF-8 LC_ALL=C.UTF-8 zanata
+$(WEBLATE_REPO):
+	git clone --depth=1 $(WEBLATE_REPO_URL) $(WEBLATE_REPO)
+	git -C $(WEBLATE_REPO) checkout $(WEBLATE_REPO_BRANCH)
 
-upload-pot: po/$(PACKAGE_NAME).pot
-	$(ZANATA_CLI) push -f --srcdir=./po --push-type=source --project-config=./po/zanata.xml $(PACKAGE_NAME).pot
+upload-pot: po/$(PACKAGE_NAME).pot $(WEBLATE_REPO)
+	cp ./po/$(PACKAGE_NAME).pot $(WEBLATE_REPO)
+	git -C $(WEBLATE_REPO) commit -m "Update source file" -- $(PACKAGE_NAME).pot
```

Comment:
Oh wow, I expected a `git add` here, I've never seen that syntax. Nice, I learned something new about git :-)